src/shared/dissect-image.c

```diff
@@ -1077,7 +1077,6 @@ int dissect_image(
                                         continue;
 
                                 designator = PARTITION_SWAP;
-                                fstype = "swap";
```

Comment:
Why is this being removed?

Reply:
This is not correct, the swap partition may be `swap` or `crypto_LUKS`.

By removing this assignment, it's set correctly near the end of this function. Without this change, the check in `add_swap` won't work.

Reply:
got it